test/test-helpers.js

```diff
@@ -0,0 +1,11 @@
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
```

Comment:
I think there may already be a place to put these.

https://github.com/BrightspaceUI/core/blob/main/tools/dom-test-helpers.js

There's the `tools` and the `test` folder. The original intention was that stuff in `test` wasn't thought to be useful to other projects so it's not published, whereas the stuff in `tools` is/was, so it is published. That could probably stand to be reviewed, but I would probably combine these with the existing ones that are published. Assuming they are moved, I'd probably rename them since the other methods don't follow this format... probably `focusWithKeyboard` and `focusWithMouse`

Reply:
Thanks Dave, done - I kept my helpers in a separate file though, because they're specifically for `@web/test-runner`, not the DOM in general (there's no way to force "keyboard mode"/"mouse mode" in the DOM directly). We may end up wanting similar helpers for `visual-diff` in the future.

Reply:
Sounds good!